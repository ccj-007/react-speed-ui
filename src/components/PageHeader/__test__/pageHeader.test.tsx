import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { PageHeader } from '../../index';

describe('test PageHeader component', () => {
  it('should render', () => {
    const { asFragment } = render(<PageHeader title={'test title'} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
