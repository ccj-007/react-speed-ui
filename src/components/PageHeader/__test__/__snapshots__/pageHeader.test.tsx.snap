// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test PageHeader component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-pageHeader-warp"
  >
    <span
      class="speed-pageHeader"
    >
      test title
    </span>
    <span
      class="speed-pageHeader-subtitle"
    />
  </div>
</DocumentFragment>
`;
